ertviz/models/plot_model.py

```diff
@@ -134,10 +134,17 @@ def repr(self):
         colors = []
         data = []
         names = []
+        realization_nums = []
         for response_name in self._data_df_dict:
             data.append(list(self._data_df_dict[response_name].values.flatten()))
             colors.append(self._colors[response_name])
             names.append(response_name)
+            realization_nums.append(
+                list(
```

Comment:
`list(foo for foo in bar)` is the same as `[foo for foo in bar]`. The first is a generator comprehension given to the `list` function to make it into a `list`, the second is a list comprehension that does it for you and doesn't require an extra indent.